src/app.js

```diff
@@ -22,8 +22,10 @@ const localisePrice = (price, locale) => {
    * Implement me!
    * Hint: .toFixed() might help...
    */
-
-  return price;
+  if(locale === 'de') return price.toFixed(2).replace('.', ',') + ' €';
+  if(locale === 'en-GB') return '€'+price.toFixed(2);
+  
+ 
```

Comment:
Here, it would help to assign a new variable for the value of `price.toFixed(2)` because it's needed for both cases. This follows the DRY principle.